### DISCUSSION TITLE
support bun runtime (loader?)

### DISCUSSION THREAD

**Discussion by nitsanavni** [avajs/ava#3113] [Ideas]
motivations: very fast, typescript just works

I was able to run tests like this:
```shell
bun run ava my-ava.spec.ts
```

together with the instructions [here](https://github.com/avajs/ava/blob/main/docs/recipes/typescript.md#for-packages-without-type-module)

howerver, these run pretty slow (more than 3 seconds for a single simple test)

**Comment by novemberborn** [comment]
`ts-node/register` can be slow, yes. I don't expect that to be `bun` related.

**Reply by nitsanavni** [reply]
I thought maybe we could somehow use bun iso ts-node. 

WDYT?

**Reply by novemberborn** [reply]
I don't know. AVA itself only uses the Node.js API, it does not include things like `ts-node`.
